src/bam/record.rs

```diff
@@ -306,6 +306,19 @@ impl Record {
         }
     }
 
+    // Delete auxiliary tag. Return false if tag doesn't exist
+    pub fn del_aux(&self, tag: &[u8]) -> bool {
```

Comment:
Well, forget the issue with the bool return. The bit-set crate does the same. Since that comes from the std actually, I think it is ok to return a bool here. Sorry for the noise.

I still think it should be `remove_aux` though.